index.bs

```diff
@@ -2012,8 +2013,7 @@ partial dictionary MLOpSupportLimits {
             input, builder.constant(input.dataType, options.minValue));
         } else {
           return builder.min(
-            builder.max(
-              input, builder.constant(input.dataType, options.minValue)),
+            builder.max(input, builder.constant(input.dataType, options.minValue)),
```

Comment:
This change and following ones seem to be made by tools/reformat-js.py.

Reply:
Ack. Seems fine to me